packages/lib/services/plugins/RepositoryApi.ts

```diff
@@ -165,24 +196,68 @@ export default class RepositoryApi {
 		}
 	}
 
-	public async search(query: string): Promise<PluginManifest[]> {
-		query = query.toLowerCase().trim();
+	public async searchAndFilter(category?: string, searchQuery?: string): Promise<PluginManifest[]> {
+		const manifests: PluginManifest[] = await this.manifests();
+		let output: PluginManifest[] = [];
+		const output2: PluginManifest[] = [];
```

Comment:
please organise the code in such a way that something like "output2" is not needed

Reply:
Done :+1: